modules/checkout/tests/src/Functional/CheckoutOrderTest.php

```diff
@@ -104,42 +110,71 @@ public function testOrderAccess() {
     $order_item->save();
     $order = Order::create([
       'type' => 'default',
-      'state' => 'in_checkout',
+      'state' => 'draft',
       'order_number' => '6',
       'mail' => 'test@example.com',
       'uid' => $user->id(),
       'ip_address' => '127.0.0.1',
       'billing_profile' => $profile,
       'order_items' => [$order_item],
+      'cart' => TRUE,
```

Comment:
Just need to add a ```store_id``` here also, since https://github.com/drupalcommerce/commerce/commit/f3e2093c7a2e6c4426e70145b0568c1f4ebc5d5d